src/Listeners/GenerateModelHelper.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Barryvdh\LaravelIdeHelper\Listeners;
+
+use Illuminate\Console\Events\CommandFinished;
+use Illuminate\Contracts\Config\Repository as Config;
+use Illuminate\Contracts\Console\Kernel as Artisan;
+
+class GenerateModelHelper
+{
+    /** @var bool */
+    public static $shouldRun = false;
+
+    /** @var \Illuminate\Contracts\Console\Kernel */
+    protected $artisan;
+
+    /** @var \Illuminate\Contracts\Config\Repository */
+    protected $config;
+
+    /**
+     * @param  \Illuminate\Contracts\Console\Kernel  $artisan
+     * @param  \Illuminate\Contracts\Config\Repository  $config
+     */
+    public function __construct(Artisan $artisan, Config $config)
+    {
+        $this->artisan = $artisan;
+        $this->config = $config;
+    }
+
+    /**
+     * Handle the event.
+     *
+     * @param  CommandFinished  $event
+     */
+    public function handle(CommandFinished $event)
+    {
+        if (!self::$shouldRun) {
+            return;
+        }
+
+        self::$shouldRun = false;
```

Comment:
Why is this necessary?

Reply:
To access the current output buffer we need to run the command on the `CommandFinished` event, where we can access the commands output.

To achieve this:
- We set the flag on the `MigrationsEnded` event
- If the flag is set we run the command
- But we need to reset the flag before that otherwise it would also trigger on the end of the generate command resulting in an infinite loop.

The best way to avoid this would be single fired events but as far as I know Laravel does not support them.

Reply:
Makes sense to me!

Can you please add this as a phpdoc comment to the `$shouldRun` property? I doubt this will be easy to figure out :)

After that I think we're good here.

Reply:
I added one